tests/integration/pkg/hooks/api-gateway.go

```diff
@@ -22,7 +23,7 @@ import (
 )
 
 const templateFileName string = "pkg/hooks/manifests/apigateway_cr_template.yaml"
-const ApiGatewayCRName string = "test-gateway"
+const ApiGatewayCRName string = "default"
```

Comment:
I would keep the separate name for the integration test gateway, maybe even add a scope by naming it `integration-test`. 
When this has a different name then it will fail early when you run it on a cluster where api-gateway is already installed. Otherwise it might lead to unexpected behaviour, if you are not using a clean api-gateway installation for the integration tests.

Reply:
I thought that as we are discouraging names other than `default`, it is not a good solution to test our integration with something different. 
And to be honest, I had a counter effect to this, meaning that my tests were failing locally since I had a `default` API Gateway CR created, and was wondering why does it fail

Reply:
Ok, good points, then let's keep default.